doc/api/tls.md

```diff
@@ -136,6 +139,8 @@ threshold is exceeded. The limits are configurable:
 The default renegotiation limits should not be modified without a full
 understanding of the implications and risks.
 
+TLSv1.3 does not support renegotiation.
```

Comment:
for consistency I think this should be prefixed with `Note:`

Reply:
Fwiw, we have (and in particular @trott) has been working towards removing `Note:` from the docs where it’s not really adding any more information.

Reply:
I've also been seeing notes removed, @vsemozhetbyt @Trott , what is current preferences?

Reply:
`Note:` style seems to be removed from the style guide in https://github.com/nodejs/node/pull/23903

Reply:
I'm happy to emphasize this some other way, or not, whatever the current style is.

Reply:
Having it as a short single-sentence paragraph by itself, the way it is now, is my preference. Setting it apart that way gives it emphasis, but isn't unduly distracting.

For things that truly must be in-your-face emphasized, bold text seems OK, but I'd recommend being very stingy about using it. Typography emphasis gets out of hand very quickly. And we've currently got a mixture of bold, italics, even all caps in a few places... 😞 

Maybe if we do the Google Summer of Docs thing (or whatever it's called), coming up with a consistent emphasis style and giving clear guidance as to what-conditions/how-often it should be used could be a good thing.

Reply:
If the main concern is consistency rather than emphasis, I'd suggest removing the other two instances of `Note:` in `tls.md` rather than adding more instances.

Lastly, none of these things I'm writing should be considered blocking. These are my opinions only and the potential for bike shedding is enormous. Just get this PR done and there will sooner or later be a PR to clean up little style things like that across docs (and, where applicable, add to the style guide and the doc-linting rules).